### PR TITLE
Two minor updates

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authentication/Win32.cs [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 
 using System;
-using System.ComponentModel;
 using System.Runtime.InteropServices;
 
 namespace Microsoft.Win32
```
Comment (PinpointTownes) [inline src/Microsoft.AspNetCore.Authentication/Win32.cs]: BTW, is this file still used? IIRC, it was used by the Twitter provider for cert pinning, but I'm pretty sure it was removed last year. /cc @Tratcher

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (Tratcher) [inline .vscode/settings.json]: Do we have this checked into any of our other repos?

Comment (troydai) [inline .vscode/settings.json]: No. But it is for the convenience of the VS code user.

Comment (Tratcher) [inline .vscode/settings.json]: @Eilon @muratg, do you want this in all of our repos? We shouldn't put it in just one, there's nothing special about this repo.

**Comment by troydai** [comment]
Remove the vs code settings. We can think about it next time.

File: src/Microsoft.AspNetCore.Authentication/Win32.cs (hunk 1 above)
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication/Win32.cs]: Remove?

**Comment by Tratcher** [comment]
Can you remove Win32.cs?

**Comment by troydai** [comment]
sure.

**Comment by troydai** [comment]
Updated

**Comment by troydai** [comment]
@Tratcher looking good?

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**